### PR TITLE
Allow more time for members to comment on PRs

### PR DESCRIPTION
**Pull request by derrell**

### CODE REVIEW

**Review by cboulanger — APPROVED** [review]

File: merge-process.md
Comment (level420) [inline merge-process.md]: I think we should use "since review started" instead of "since the call for votes"

File: merge-process.md
Comment (level420) [inline merge-process.md]: We should use the github review wording here "approval" and "request for changes".
And we could explain that adding a change request blocks merging.

**Review by level420 — CHANGES_REQUESTED** [review]

**Review by level420 — APPROVED** [review]

**Review by cboulanger — APPROVED** [review]

**Review by hkollmann — APPROVED** [review]

**Review by level420 — CHANGES_REQUESTED** [review]
I noticed just now that the passage `at least two business days have passed since the call for votes, and two assenting votes have been cast with no dissenting votes` has been removed completely as a positive requirement "signal" for merging.

From my point of view we still need something similar, otherwise we will have PRs with 4 approvals which never get merged because the core team members are busy or simply forgot about the review.

What about: 
`at least 5 business days have passed since review request, and 3 assenting votes (approvals) have been cast with no dissenting votes (request for changes)`

**Comment by derrell** [comment]
If we can't get 5 approvals, I'm not sure a PR should be merged. If we happen to hit a time when many core members are away, the PR may just have to wait a week or more before merging, but I still think it's worthwhile allowing people to comment even if on holiday for a while. It's not like (in the vast majority of cases), people can't use the changes from the PR in their own development during that period...

**Comment by level420** [comment]
@derrell OK! convinced.

**Review by level420 — APPROVED** [review]

**Review by cboulanger — APPROVED** [review]

**Review by oetiker — APPROVED** [review]

**Review by johnspackman — APPROVED** [review]

**Review by cajus — APPROVED** [review]

**Pull request merged**
